src/app/components/feed/jh-feed.directive.js

```diff
@@ -116,6 +119,20 @@
         }
       }
 
+      function isEditable(){
```

Comment:
The name of this function is quite confusing. Functions called `isWhatever` are expected to be checks (with no effect) returning a boolean value.

I would try something closer to `makeNameEditable` (just a possibility).